buildSrc/src/main/groovy/com.google.api-ads.java-conventions.gradle

```diff
@@ -153,7 +153,9 @@ dependencies {
   implementation 'com.google.api:gax'
   implementation 'com.google.api:gax-grpc'
   implementation 'com.google.protobuf:protobuf-java'
-  implementation 'io.grpc:grpc-all'
+  implementation 'io.grpc:grpc-stub'
```

Comment:
Does `google-cloud-shared-dependencies` actually already cover these?

https://github.com/googleapis/java-shared-dependencies/blob/main/first-party-dependencies/pom.xml

Reply:
Yep, it imports [gRPC BOM](https://github.com/googleapis/java-shared-dependencies/blob/37395d148ab5cb4ccf9bbe05e21656e81ec53a89/first-party-dependencies/pom.xml#L75-L80) which declares [grpc-protobuf and grpc-stub](https://repo1.maven.org/maven2/io/grpc/grpc-bom/1.42.1/grpc-bom-1.42.1.pom).